docs_src/design/adr/0027-URIs for Files.md

```diff
@@ -0,0 +1,72 @@
+# URIs for Files ADR
+### Submitters
+- Lenny Goodell (Intel)
+
+## Change Log
+- [pending ](https://github.com/edgexfoundry/edgex-docs/pull/949) (2023-02-01)
+
+## Referenced Use Case(s)
+- [URIs for Files UCR](https://docs.edgexfoundry.org/3.0/design/ucr/URIs-for-Files/)
+
+## Context
+Currently, in the Levski and earlier releases services can only load configuration, units of measurements, device profiles, device definitions, provision watches, etc. from the local file system. As outline in the reference UCR, there is a need to be able to load files from a remote locations using a URIs to specify the locations. 
+
+## Proposed Design
+This ADR proposes a new helper function for loading files be added to `go-mod-bootstrap`. This function will provide the logic for loading a file either from local file system (as is today) or from a remote location. As stated in the UCR, only **HTTP** and **HTTPS** URIs will be supported. For **HTTPS**, certificate validation will be performed using the system's built-in trust anchors. The docker images for all services will have the CA certs installed as is done here in App Service Configurable's [Dockerfile](https://github.com/edgexfoundry/app-service-configurable/blob/v2.3.0/Dockerfile#L46).
+
+### Authentication 
+
+While not recommended, users will be able to specify **basic-auth** (`<username>:<password>@`) in the URI in plain text. In order to provide a secure way for users to specify credentials, the URI scheme will be expanded to allow specifying a Secret Name from the service's Secret Store. This will take the form  `[secretName: <secret-name>]@`. The type of authentication as well as the credentials will be contained in the secret data specified by the Secret Name. Three types of authentication will be supported, which are `usernamepassword`, `apikey` and `bearertoken`. 

```

Comment:
There's a space between secretName: and <secret-name> - is that valid in a URL, even if inside the square brackets?
The square brackets here are literal and don't have the usual meaning of "optional parameter", right?

Reply:
Yes, the space probably should be removed.
`[secretName: <secret-name>]@` is our own addition and will always be replaced or removed so will never be in the actual URL that is used.

Reply:
Removed space